packages/backend-test-utils/src/next/wiring/TestBackend.ts

```diff
@@ -47,11 +53,19 @@ export interface TestBackendOptions<
   features?: BackendFeature[];
 }
 
+const defaultServiceFactories = [
+  rootLoggerFactory(),
+  loggerFactory(),
+  lifecycleFactory(),
```

Comment:
I'd totally be up for a config service with an empty config, and perhaps a database service (that creates a sqlite database automagically if there's no backend.database key), ... and maybe some more.

Could be in a separate PR of course, if at all. It's a bit of a promise to make of course, to have them ever-present with a certain behaviour.

Reply:
Oh what, didn't realize these changes where part of this 😅. Either way, it's covered in #13082, yep separate PR